test/extended/router/scoped.go

```diff
@@ -242,6 +242,7 @@ func waitForRouterOKResponseExec(ns, execPodName, url, host string, timeoutSecon
 	cmd := fmt.Sprintf(`
 		set -e
 		for i in $(seq 1 %d); do
+			rc=0
```

Comment:
Would it be worth backporting https://github.com/openshift/origin/pull/23614/commits/743b473afe412ae14061e3ee741c472962df1fe7 as well in order to keep the code in synch?

Reply:
I think any test stability back ports would be welcome

Reply:
https://github.com/openshift/origin/commit/743b473afe412ae14061e3ee741c472962df1fe7 is older, so it would make sense to backport it first.  Would you like a separate PR, or will you incorporate it into this PR?

Reply:
(Presumably we'd want all the commits in https://github.com/openshift/origin/pull/23614.)